docs/docs/specs/v0.2/shell.md

```diff
@@ -0,0 +1,14 @@
+
+# DXOS Shell 
+
+## Scenarios
+
+1. Users can create new HALO identities on supported devices
+2. Users can link devices to existing HALO identities on other devices
+3. Users can sign-in to applications with their HALO identity
```

Comment:
what does this mean?

Reply:
this is the `useIdentity()` thing where a 3P app can get an identity and shell takes over for the sign in

Reply:
maybe better phrased `applications can obtain the user's HALO identity using the SDK` or something

Reply:
yeah, I was trying to figure out how it was different from point 4